mapper/orm/src/main/java/org/hibernate/search/mapper/orm/loading/impl/AbstractHibernateOrmLoadingStrategy.java

```diff
@@ -32,7 +36,8 @@
 
 	@Override
 	public HibernateOrmQueryLoader<E, I> createQueryLoader(
-			Collection<PojoRawTypeIdentifier<? extends E>> targetEntityTypes) {
+			Collection<PojoRawTypeIdentifier<? extends E>> targetEntityTypes,
```

Comment:
I think you can remove this parameter? You have all the information you need with `typeContexts`.

Note that `targetsAllConcreteSubTypes` expects a `Collection<?>`, not a `Collection<PojoRawTypeIdentifier<?>>`, so you can pass `typeContexts` instead.